src/plugins/index_pattern_management/public/components/index_pattern_table/index_pattern_table.tsx

```diff
@@ -197,25 +193,21 @@ export const IndexPatternTable = ({ canSave, history }: Props) => {
   }
 
   return (
-    <EuiPageContent data-test-subj="indexPatternTable" role="region" aria-label={ariaRegion}>
-      <EuiFlexGroup justifyContent="spaceBetween">
-        <EuiFlexItem grow={false}>
-          <EuiTitle>
-            <h2>{title}</h2>
-          </EuiTitle>
-          <EuiSpacer size="s" />
-          <EuiText>
-            <p>
-              <FormattedMessage
-                id="indexPatternManagement.indexPatternTable.indexPatternExplanation"
-                defaultMessage="Create and manage the index patterns that help you retrieve your data from Elasticsearch."
-              />
-            </p>
-          </EuiText>
-        </EuiFlexItem>
-        <EuiFlexItem grow={false}>{createButton}</EuiFlexItem>
-      </EuiFlexGroup>
-      <EuiSpacer />
+    <div data-test-subj="indexPatternTable" role="region" aria-label={ariaRegion}>
```

Comment:
Nit: you can use `title` for the `aria-label` here because they should always match and then you can get rid of a duplicate string.

```suggestion
    <div data-test-subj="indexPatternTable" role="region" aria-label={title}>
```